libraries/AP_ADSB/AP_ADSB.cpp

```diff
@@ -462,8 +462,8 @@ void AP_ADSB::send_adsb_vehicle(const mavlink_channel_t chan)
 
 void AP_ADSB::send_dynamic_out(const mavlink_channel_t chan)
 {
-    AP_GPS gps = _ahrs.get_gps();
-    Vector3f gps_velocity = gps.velocity();
+    const AP_GPS &gps = _ahrs.get_gps();
```

Comment:
this is not the only place actually. See "AP_ADSB: fix getting a copy rather than a reference" and "global: use static method to construct AP_GPS".